README.md

```diff
@@ -71,6 +71,49 @@ With `flox` you can:<br/>
 - .. more (point to the website)
 -->
 
+# Flox vs Docker:
+Before demonstrating the difference, let's clarify what Docker is. Historically, the fundamental operation of Docker is based on `chroot`, a system call introduced in Version 7 Unix. This system call changes the root directory for the current process, making the specified directory appear as the root (`/`) for any process running within it. Later, Linux adopted this feature, which became known as Linux Containers. In 2008, Docker was built based on this concept. Simply put, a container is a running process that uses virtualization-based mechanisms, but each of these containers perceives its own environment as if it were the host system. Given this, you can see Flox is different than Docker.
+
+# Flox vs Homebrew:
+
+This document compares **Homebrew** and **Flox**, two popular package managers used in modern development environments. Each has its own strengths and is suited to different use cases. Here's a side-by-side comparison of their key features:
```

Comment:
>This document

The README? Perhaps this section of the document, but if it's doing that, it doesn't have to be called out as it doing that.